# State is used to represent the state of the StateMachine
# State  is defined as:
#    0 - float charge (last 10%)
#    1 - charging
#    2 - not charging
int8 currentState